Index: src/mainwindow.cpp
===================================================================
--- src/mainwindow.cpp.orig	2017-11-04 16:11:25.163538614 +0100
+++ src/mainwindow.cpp	2017-11-04 16:17:17.798846928 +0100
@@ -2547,9 +2547,9 @@
 
 void MainWindow::showUpgradePrompt()
 {
-    QAction* action = new QAction(tr("Click here to check for a new version of Shotcut."), 0);
-    connect(action, SIGNAL(triggered(bool)), SLOT(on_actionUpgrade_triggered()));
-    showStatusMessage(action, 30 /* seconds */);
+    //QAction* action = new QAction(tr("Click here to check for a new version of Shotcut."), 0);
+    //connect(action, SIGNAL(triggered(bool)), SLOT(on_actionUpgrade_triggered()));
+    //showStatusMessage(action, 30 /* seconds */);
 }
 
 void MainWindow::on_actionRealtime_triggered(bool checked)
@@ -2908,7 +2908,7 @@
 void MainWindow::on_actionUpgrade_triggered()
 {
     showStatusMessage("Checking for upgrade...");
-    m_network.get(QNetworkRequest(QUrl("http://check.shotcut.org/version.json")));
+    m_network.get(QNetworkRequest(QUrl("file:///usr/share/shotcut/version.json")));
 }
 
 void MainWindow::on_actionOpenXML_triggered()
@@ -3040,7 +3040,8 @@
                     m_upgradeUrl = json.object().value("url").toString();
                 showStatusMessage(action, 10 /* seconds */);
             } else {
-                showStatusMessage(tr("You are running the latest version of Shotcut."));
+                showStatusMessage(tr("Use dnf for updates."));
+
             }
             reply->deleteLater();
             return;
Index: src/src.pro
===================================================================
--- src/src.pro.orig	2017-11-04 16:22:22.359946204 +0100
+++ src/src.pro	2017-11-04 16:25:41.796318201 +0100
@@ -326,7 +326,7 @@
 LIBS += -lCuteLogger -lmvcp -lpthread
 
 isEmpty(SHOTCUT_VERSION) {
-    !win32:SHOTCUT_VERSION = $$system(date "+%y.%m.%d")
+    !win32:SHOTCUT_VERSION = $$_VSTRING
      win32:SHOTCUT_VERSION = adhoc
 }
 DEFINES += SHOTCUT_VERSION=\\\"$$SHOTCUT_VERSION\\\"
